tsdb/raw.go

```diff
@@ -93,6 +93,7 @@ func (e *RawExecutor) execute(out chan *models.Row) {
 	// Keep looping until all mappers drained.
 	var err error
 	for {
+		timeout := time.After(30 * time.Second)
```

Comment:
This is intended for if they don't read off a cursor for an extended amount of time.  In general, the calling code should be reading off as fast as it can, and 30 seconds would be an eternity for it to not read right?  I may have the timeout in the wrong place in this specific piece of code (as stated above).  Thoughts?